LICENSE

```diff
@@ -186,8 +186,9 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
-
+   Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+   Copyright [2016-2020] EMBL-European Bioinformatics Institute
+   
```

Comment:
Based on my understanding, the LICENSE file is the Apache 2.0 template, and these two lines should go in the NOTICE file (which we have updated already):
> To apply the Apache License to your work, attach the following boilerplate notice, with the fields enclosed by brackets "[]" replaced with your own identifying information.

```suggestion
   Copyright [yyyy] [name of copyright owner]
   
```

Reply:
Makes sense ... did not check the NOTICE file!!